scipy/signal/tests/test_signaltools.py

```diff
@@ -1536,7 +1536,11 @@ def test_method(self, dt):
             y_fft = correlate(a, b, method='fft')
             y_direct = correlate(a, b, method='direct')
 
-            assert_array_almost_equal(y_r, y_fft, decimal=self.equal_tolerance_fft(y_fft.dtype))
+            if dt == np.float128:
```

Comment:
This causes CI failures:
```
AttributeError: module 'numpy' has no attribute 'float128'
```
look for other tests on how to handle this, IIRC we have some (probably check with `hasattr` first). `float128`, `float96`, `complex256` are not guaranteed to exist